web_src/js/components/RepoActionView.vue

```diff
@@ -90,14 +102,19 @@ const sfc = {
     runIndex: String,
     jobIndex: String,
     actionsURL: String,
+    artifactsTitle: String,
   },
 
   data() {
     return {
+      ansiToHTML: new AnsiToHTML({escapeXML: true}),
+      artifactsTitle: this.artifactsTitle,
```

Comment:
Hmm ... it seems not right to do so in Vue? I don't understand why `var: this.var` here ....

Reply:
But `this` is already in `Vue` here? Why need to assign a var to itself